### DISCUSSION TITLE
Ability to programmatically update your linked IP with ADGuard DNS

### DISCUSSION THREAD

**Discussion by CaptainComeback** [AdguardTeam/AdGuardHome#5628] [Ideas]
I am a huge fan of adguard home and I also use Adguard DNS. Currently I use both so I can have a secure tunnel for my DNS requests and my internet provider does not like this and they constantly push a new IP address daily.  There is a link to programmatically update your linked IP from the adguard DNS website and I was wondering if there is or how hard it would be to build that into adguard home? Currently I try to check regularly and it seems like every day I have a new IP. Keep up the good work and any suggestions would be great.

**Comment by Disqu3-mirOir-qt** [comment]
in you,r case DDNS can help you 

 https://www.fortinet.com/resources/cyberglossary/dynamic-dns

**Comment by YBS-PC** [comment]
i use  a link to programmatically update linked IP in DNS allowlists and my linked ip is updated every hour

**Reply by CaptainComeback** [reply]
I just added the programmatic link URL to the dns update lists so every time it checks for updates it will hit that URL. Thanks for the reply.
